frontend/admin/src/js/components/home/HomePage.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { HomeIcon, LoginIcon } from "@heroicons/react/outline";
+import CardLink from "@common/components/CardLink";
+import PageHeader from "@common/components/PageHeader";
+import { navigate } from "@common/helpers/router";
+
+import { AuthContext } from "../AuthContainer";
+import { useAdminRoutes } from "../../adminRoutes";
+
+const HomePage: React.FC = () => {
+  const intl = useIntl();
+  const { loggedIn } = React.useContext(AuthContext);
+  const paths = useAdminRoutes();
+
+  /**
+   *   Note: I'm not sure if this is best practice
```

Comment:
> Honestly, I don't know either. I feel like I would still want to show the landing page and allow the user to navigate somewhere else themselves. But what would the button look like? @Jerryescandon ?

Im not sure I follow on this one, can anyone zoom me to explain?

Reply:
I suppose option B for the reroute is to have it in the router so the user doesn't hit this page at all.  Basically, the opposite of hiding pages for non-admins.  I guess that would be a little cleaner and quicker since they don't get the flash of this page before disappearing again?

Reply:
Lol, funny you should mention that. I just thought of that and am about 1-2 mins way from pushing that change up now. Just testing it

Reply:
Okay, so [rearrange auth, pass ctx to routes for redirects](https://github.com/GCTC-NTGC/gc-digital-talent/pull/2675/commits/b27cf1ff480027ff0db0439608066460df507020) should resolve this.

I had to move the `AuthProvider` up one level so we can check the user state and pass it to the router.

Reply:
Head's up @mnigh.  I think you're working on the admin auth container right now, too.

Reply:
Okay, the link has been updated to the `apiRoutes.login`. I noticed that it was using `navigate` which was causing the 404. Adding the `external` prop properly sends the user to the login page.

Resolved by [fix login link to use apiRoutes](https://github.com/GCTC-NTGC/gc-digital-talent/pull/2675/commits/a5b58e1d3fc2326250baa489f844f7f008e4418f)